packages/suite-desktop/src-electron/electron.ts

```diff
@@ -209,24 +208,32 @@ app.on('browser-window-focus', (_event, win) => {
     }
 });
 
-// listen the channel `message` and resend the received message to the renderer process
-ipcMain.on('message', (event, message) => {
-    event.sender.send('message', message);
-});
-
-ipcMain.on('start-bridge', async (_event, devMode?: boolean) => {
+ipcMain.on('bridge/start', async (_event, devMode?: boolean) => {
     try {
         await runBridgeProcess(devMode);
     } catch (error) {
         // TODO: return error message to suite?
     }
 });
 
-ipcMain.on('restart-app', () => {
+ipcMain.on('app/restart', () => {
     app.relaunch();
     app.exit();
 });
 
-ipcMain.on('oauth-receiver', (_event, message) => {
-    mainWindow.webContents.send('oauth', { data: message });
+ipcMain.on('app/focus', () => {
```

Comment:
Where's the event for this being triggered?

Reply:
at the time you were reading this it was triggerd from getOauthCode() method in suite/utils/metadata. But we discussed with szymon that we can actually trigger this from the electron layer and get rid of unneeded indirection.